src/clojure/nrepl/cmdline.clj

```diff
@@ -3,11 +3,25 @@
   e.g. REPL-y for a proper command-line nREPL client @
   https://github.com/trptcolin/reply/"
   {:author "Chas Emerick"}
-  (:require [clojure.java.io :as io]
+  (:require [clojure.edn :as edn]
+            [clojure.java.io :as io]
             [nrepl.core :as repl]
             [nrepl.ack :refer [send-ack]]
             [nrepl.server :refer [start-server]]))
 
+(defn exit
```

Comment:
Please, add some docstring to each method. I'm very committed to documenting everything. :-)